src/Client/Modals/ModalElements.fs

```diff
@@ -14,6 +14,20 @@ open Fable.React.Helpers
 
 type ModalElements =
 
+    static member LogicContainer (children: ReactElement list) =
```

Comment:
Move to `GenericComponents.fs`

Reply:
Now that i think about it. Why do you need this in a modal? this should only be used for sidebar logic groups?

Reply:
I grouped all elements that were used by SeletiveTemplateFromDB.fs and SelectiveImporModal.fs into the ModalElements.fs.
Origininally I rebuild the whole template selection as a modal but have undone it.